sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -408,4 +406,5 @@ Compute absolute value of a `bfloat16`.
 |3|2021-08-18|Alexey Sotkin |Remove `uint16_t` constructor
```

Comment:
This comment is for the math function lines like:

```
namespace sycl::ext::oneapi::experimental {

template <typename T>
T fma(T a, T b, T c);
} // namespace sycl::ext::oneapi::experimental
```

I think it would be better to show the synopsis like this, with two overloads:

```
namespace sycl::ext::oneapi {

bfloat16 fma(bfloat16 a, bfloat16 b, bfloat16 c);

template<size_t N>
marray<bfloat16, N> fma(marray<bfloat16, N> a, marray<bfloat16, N> b, marray<bfloat16, N> c);

} // namespace sycl::ext::oneapi
```

This makes it clear that it is OK to pass an argument that can be implicitly converted to `bfloat16`.  See the discussion in https://gitlab.khronos.org/sycl/Specification/-/issues/278

Reply:
This can be done in a later PR.

Reply:
OK, done in this PR.